### DISCUSSION TITLE
Add ability to export and import evaluators between projects

### DISCUSSION THREAD

**Discussion by j10sanders** [langfuse/langfuse#3970] [Ideas]
### Describe the feature or potential improvement

Similar to https://github.com/orgs/langfuse/discussions/1696

I've created numerous evaluators in my non-prod project, and re-creating them in the production project is going to be a hassle. I would love to be able to copy evaluators between projects.

**Comment by marcklingen** [comment]
thanks for sharing! agree that this workflow needs to be easier. currently evaluating whether in general having native support for different environments within a single langfuse tracing project would be the simpler solution here going forward

**Reply by twerkmeister** [reply]
Hey @marcklingen, what's the latest on this? I am preparing an enterprise client to start using langfuse. I know which software they are going to use (rasa) so I am showcasing to them already how everything would look like with a demo bot. Would be super useful to be able to export and import the evaluators already configured so that they can hit the ground running and work from that and don't need to create the entire LLM as a judge observability from scratch which is their main interest at this point.

**Reply by twerkmeister** [reply]
In this case it wouldn't be within a single tracing project. I am just showcasing in a small demo project and they are going to self host.

**Reply by twerkmeister** [reply]
Just also wondering why you can export pretty much everything but not the evaluators? Seems odd
